vendor/node-usb-native/src/combined.cpp

```diff
@@ -9,11 +9,11 @@
 #include <nan.h>
 
 extern "C" {
-	void init_serialport(v8::Handle<v8::Object> target);
+	void init_serialport( v8::Local<v8::Object> target);
```

Comment:
Extra space included here, I think this is due to copied from the Arduino repo, please also fix it here.

Reply:
Fixed.